Automattic/woocommerce-payments
# Replace payment method logos component with the shared component from `@woocommerce/onboarding`

Fixes #9002

#### Changes proposed in this Pull Request
This PR replaces the existing duplicate payment method logos component with the shared component from `@woocommerce/onboarding`, implemented in the WooCommerce core. By using the shared component, we ensure consistency across the platform and reduce redundancy in the codebase.

<img width="691" alt="Screenshot 2024-07-29 at 17 06 11" src="https://github.com/user-attachments/assets/77d523cd-fe87-4160-9b48-76acb56d5ba4">

#### Testing instructions
- Delete any onboarded account and clear cache.
- Navigate to `Payments` page.
- Ensure the Payment Methods logos component is displayed correctly.


-------------------


**Post merge**
